vnext/Microsoft.ReactNative/Modules/Animated/NativeAnimatedModule.h

```diff
@@ -68,6 +68,7 @@ class NativeAnimatedModule final : public facebook::xplat::module::CxxModule {
   auto getMethods() -> std::vector<Method> override;
 
   void CreateAnimatedNode(int64_t tag, const folly::dynamic &config);
+  void GetValue(int64_t tag, const Callback &endCallback);
```

Comment:
can be noexcept?

Reply:
TryGetScalar is noexcept, so I guess you're right :)